src/SignalR/common/Http.Connections/src/HttpConnectionDispatcherOptions.cs

```diff
@@ -17,7 +17,7 @@ public class HttpConnectionDispatcherOptions
     {
         // Selected because this is the default value of PipeWriter.PauseWriterThreshold.
         // There maybe the opportunity for performance gains by tuning this default.
-        private const int DefaultPipeBufferSize = 32768;
+        private const int DefaultBufferSize = 65536;
```

Comment:
I see you

Reply:
Do we also want to increase the max message size?
https://github.com/dotnet/aspnetcore/blob/8b30d862de6c9146f466061d51aa3f1414ee2337/src/SignalR/server/Core/src/HubOptionsSetup.cs#L23

Reply:
We don't have to.